app/components/safe_area_view/safe_area_view.ios.js

```diff
@@ -119,7 +121,7 @@ export default class SafeAreaIos extends PureComponent {
     };
 
     onSafeAreaInsetsForRootViewChange = ({safeAreaInsets}) => {
-        if (EphemeralStore.safeAreaInsets.portait !== null && EphemeralStore.safeAreaInsets.landscape !== null) {
+        if (EphemeralStore.safeAreaInsets[PORTRAIT] !== null && EphemeralStore.safeAreaInsets[LANDSCAPE] !== null) {
```

Comment:
this wasn't really neccesary, only when comparing the strings, as people can easily make mistakes. By having a constant, the compiler will warn you that you misspelled it.

once you are using dot notation, there is no real need for that. But I also don't think you need to change it back. You could have saved some effort, sorry if I wasn't clear on my request.